tests/Fixtures/Integration/misc/method_separation,indentation_type.test

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Integration of fixers: method_separation, indentation_type
+--RULESET--
+{"class_attributes_separation": true, "indentation_type": true}
```

Comment:
Not matching description.

Reply:
good catch, I also created it in the wrong directory and didn't update the FixerFactoryTest :man_facepalming: